USAGE.md

```diff
@@ -122,3 +122,42 @@
 # plot pdf of exponential distribution 
 >>> exponential.plot_bar_pdf() 
 ```
+
+## For Gamma Distribution
+```
+>>> from probdists import Gamma
+
+# By default Gamma will create a gamma dist. with k=2, theta=2 and won't fit to data
+# This is useful if you want a simple Gamma distribution to play around with the pdf
+# HOWEVER: It will not make use of data passed into the distribution
+>>> gamma = Gamma()
+
+# To use of the sample data or your own data, and approximate a gamma fit to that data:
+>>> gamma = Gamma(fit=True, data_file='demo_gamma_data')
+# for your own file, replace 'demo_gamma_data' with 'my_data_file.txt' 
+# Ensure there is no extra whitespace at end of file
```

Comment:
This is being taken care of in another PR, where the read_data_file() function is being improved to be more robust and deal with such cases